app/common/services/strataService.js

```diff
@@ -197,6 +197,25 @@ angular.module('RedhatAccess.common').factory('strataService', [
                     return deferred.promise;
                 }
             },
+            articles: {
+                get: function (uri) {
+                    var deferred = $q.defer();
+                    var splitUri = uri.split('/');
+                    uri = splitUri[splitUri.length - 1];
```

Comment:
Correct me if I am wrong, but this code seems to parse the article ID out of the URI. Why are you not passing the ID of the article instead? Seems like a hassle to pass the URI just to extract the ID out of it.